test.js

```
@@ -16,3 +16,16 @@ test('should not verify a wrong password with pbkdf2', async t => {
   t.true(typeof hash === 'string');
   t.false(await pify(m.verify)(hash, 'hello world'));
 });
+
+test.serial('invalid password with pbkdf2', async t => {
```

should throw an error trying to hash a non valid string